docs/src/components/Utils/index.ts

```diff
@@ -1,10 +1,26 @@
+import { css } from '@xstyled/styled-components';
+
 export { ApiTable } from './ApiTable';
 export { ColorBlurb } from './ColorBlurb';
 export { ComponentInfo } from './ComponentInfo';
 export { FormatTypes } from './FormatTypes';
-export { PositionGrid } from './PositionGrid';
+// Taking the shotgun approach here because a Type is being exported. Gatsby can handle the wildcard
+// but not if the export is named
+export * from './Responsive';
```

Comment:
Did you try modifying the `anchor/docs/src/typings.d.ts` file with the named exports?

Reply:
Found a solution that imports the Type then explicitly declares the export. Tested in the docker test build along with gatsby dev w/no errors.